src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -86,6 +86,11 @@
  */
 #define LOG_PER_TUPLE_AMOUNT 1000000
 
+#define UNDISTRIBUTE_TABLE_CASCADE_HINT \
+	"Use cascade option to undistribute other relations involved in " \
+	"a foreign key relationship with \"%s\" by executing SELECT " \
+	"undistribute_table(%s, cascade=>true)"
```

Comment:
Let's improve the hint with single quotes and make sure that the table name is accurately quoted so that the user can simply copy & paste the hint and can proceed